sample/docker-compose.local-sample.yml

```
@@ -3,8 +3,8 @@ version: '2'
 services:
   nginx:
     ports:
-    - ${SERVER_HTTP_PORT}
+    - ${SERVER_HTTP_PORT}:80
```

Only for dynamic port assign. Use `${SERVER_HTTP_PORT}:80` in `./docker/docker-compose.local.yml`
```suggestion
    - ${SERVER_HTTP_PORT}
```